Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'validates_email_format_of', '~> 1.6', '>= 1.6.3'
 gem "geocoder"
 # Rails 4 support is a work in progress so requires tracking master
 gem 'activeadmin', '~> 1.0.0.pre2'
-gem "devise"
+gem "devise", '~> 3.5'
```

Comment:
I just tried the update myself and got:

```
Gem::InstallError: devise requires Ruby version >= 2.1.0.
An error occurred while installing devise (4.2.0), and Bundler cannot continue.
Make sure that `gem install devise -v '4.2.0'` succeeds before bundling.
```

So we know what the problem is :smile: I'll add a note myself.